content/docs/intro/cloud-providers/docker/_index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Docker
+meta_desc: This page provides an overview of the Docker Provider for Pulumi.
+menu:
+  intro:
+    parent: cloud-providers
+    identifier: clouds-docker
+    weight: 5
+
+aliases: ["/docs/reference/clouds/docker/"]
+---
+
+<img src="/logos/tech/digitalocean.svg" align="right" class="h-16 px-8 pb-4">
```

Comment:
Switch to using the docker logo rather than digitalocean.

Reply:
This is fixed - oversight